mcs/class/System.Drawing/Test/System.Drawing.Drawing2D/TestMatrix.cs

```diff
@@ -40,8 +40,8 @@ namespace MonoTests.System.Drawing.Drawing2D
 	public class MatrixTest {
 
 		private Matrix default_matrix;
-		private Rectangle rect;
-		private RectangleF rectf;
+		private Rectangle rect = default (Rectangle);
```

Comment:
OK, done. Looks like default_matrix is an unused field as well, so I removed that, too.